modules/acquia_cms_event/composer.json

```diff
@@ -4,6 +4,7 @@
     "license": "GPL-2.0-or-later",
     "type": "drupal-module",
     "require": {
+        "drupal/acquia_cms_common": "^1.9 || ^2.1 || ^3.1",
```

Comment:
**FYI:** The dependency on the `acquia_cms_common` module will be removed for content model to function independently in separate PR.
Same apply for all other content type modules.